### ISSUE TITLE
autocomplete popup missing

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#127]
Reported by @ammarawaisi : 

>  In previous versions, typing / in the prompt input triggered an autocomplete popup for commands. This is no longer appearing. While I can still manually type /describe, the visual cue is gone.

this will be prioritized and fixed in sprint 5, since it affects accessibility
